src/components/Swatch/Swatch.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+class Swatch extends Component {
+  constructor() {
+    super()
+    this.state = {
+      locked: false,
+    }
```

Comment:
I know that we talked about it, but it's nice to see that we are breaking up the responsibility for each swatch to be it's own 👍